### DISCUSSION TITLE
static_export configuration

### DISCUSSION THREAD

**Discussion by VovaParamonov** [jaredpalmer/razzle#1689] [Q&A] [answered]
According to [this article](https://razzlejs.org/docs/static-export), I have to import a `renderApp` from the server.js file. (but this file does not contain such an export...) How should I understand this?

**Comment by fivethreeo** [comment, marked as answer]
yarn create razzle-app --example basic my-app

or

yarn create razzle-app --example with-react-router my-app
for a full example
